src/Akka.Persistence.Linq2Db.HelperLib/Akka.Persistence.Linq2Db.HelperLib.csproj

```diff
@@ -1,16 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <Import Project="..\common.props" />
 
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
+        <LangVersion>10</LangVersion>
     </PropertyGroup>
 
     <ItemGroup>
       <ProjectReference Include="..\Akka.Persistence.Sql.Linq2Db\Akka.Persistence.Sql.Linq2Db.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="FluentMigrator" Version="$(FluentMigratorVersion)" />
+      <PackageReference Include="FluentMigrator" Version="3.3.1" />
```

Comment:
Probably should still use the sync variable here

Reply:
All done